product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -107,6 +107,58 @@ def test_pricelist_min_quantity(self):
             self.pricelist.get_product_price(self.product, 10, False), 50,
         )
         self.pricelist.item_ids[0].no_supplierinfo_min_quantity = True
+        # As the sequence of both of the supplieinfo is the same the next field to
+        # order is the min_qty (desc), so the price must be the one with the higher
+        # min_qty no mather if the qty is lower that the min_qty.
+        self.assertAlmostEqual(
+            self.pricelist.get_product_price(self.product, 1, False), 50,
+        )
+        self.assertAlmostEqual(
+            self.pricelist.get_product_price(self.product, 5, False), 50,
+        )
+
+    def test_pricelist_no_min_quantity(self):
+        """Testing the validations of the `_select_seller_without_min_qty` method.
+        As the sequence of both of the supplieinfo in the product is the same the
+        next field to order is the min_qty (desc), so the price must be the one
+        with the higher min_qty (the one with the price 50), but we are going to test
+        the following cases in the supplierinfo with the higher min_qty:
+        - The start date is tomorrow (not current).
+        - The end date is yesterday (not current).
+        - The supplierinfo is current, but there is a 'Vendor' set in the field
+        `filter_supplier_id` of the item in the pricelist.
+        """
+        product = self.product
+        tomorrow = date.today() + timedelta(days=1)
+        product.seller_ids[0].date_start = tomorrow
+        self.pricelist.item_ids[0].no_supplierinfo_min_quantity = True
+        # Case 1: The start date is not a current one so it will takes the next one.
```

Comment:
```suggestion
        # Case 1: The start date is not a current one so it will take the next one.
```

or even better, "will be taken"

Reply:
Done